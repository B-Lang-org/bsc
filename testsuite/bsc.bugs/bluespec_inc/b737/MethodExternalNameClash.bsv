interface IFC;
   method Bit#(32) getVal();
endinterface

(* synthesize *)
module sysMethodExternalNameClash (IFC);
   Bit#(32) getVal = 0;
   method Bit#(32) getVal();
      return 17;
   endmethod
endmodule

