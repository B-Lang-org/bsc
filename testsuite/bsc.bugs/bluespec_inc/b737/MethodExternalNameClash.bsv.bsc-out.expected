checking package dependencies
Error: "MethodExternalNameClash.bsv", line 8, column 4: (T0011)
  Declaration of `getVal' conflicts with previous declaration at
  "MethodExternalNameClash.bsv", line 7, column 13
